### PR TITLE
UI : Styling Improvements

### PR DESCRIPTION
**Pull request by aniketkatkar97**
- Made styling improvements
- Fixed bug with the side navigation bar items expansion

Screenshot 2022-10-18 at 9 06 56 PM
Screenshot 2022-10-18 at 9 07 10 PM
Screenshot 2022-10-18 at 9 07 20 PM
Screenshot 2022-10-19 at 11 08 32 AM

### CODE REVIEW

**Comment by ShaileshParmar11** [comment]
The build is failing for netlify!

**Comment by Sachin-chaurasiya** [comment]
Hello @aniketkatkar97 , Can we convert the javascript files to typescript files?

**Comment by aniketkatkar97** [comment]
> The build is failing for netlify!
> 
> image

Fixed the fails

File: components/TopNav/TopNav.tsx [hunk 1]
```diff
@@ -24,15 +24,15 @@ export default function TopNav() {
...
-            <option value="0.10.0">V 0.10.0</option>
-            <option value="0.10.1">V 0.10.1</option>
-            <option value="0.11.0">V 0.11.0</option>
-            <option value="0.11.1">V 0.11.1</option>
-            <option value="0.11.2">V 0.11.2</option>
-            <option value="0.11.3">V 0.11.3</option>
-            <option value="0.11.4">V 0.11.4</option>
-            <option value="0.12.0">V 0.12.0</option>
-            <option value="0.12.1">V 0.12.1</option>
+            <option value="0.10.0">v 0.10.0</option>
```
Comment (chirag-madlani) [inline components/TopNav/TopNav.tsx:27]: Why hardcoding? Instead try to maintain a version array instead?

Comment (aniketkatkar97) [reply components/TopNav/TopNav.tsx]: My bad, will take care of this

**Review by chirag-madlani — APPROVED** [review]

**Pull request merged**